README.rst

```diff
@@ -272,16 +272,17 @@ To create a Rackspace OnMetal slave to serve this purpose:
 * Log into https://mycloud.rackspace.com,
 * Create Server > OnMetal Server,
 * Give the server an appropriate name,
-* Choose the following options: Image: OnMetal - Fedora 20 (Heisenbug), Flavor: OnMetal Compute, An SSH key you have access to
+* Choose the following options: Image: OnMetal - Fedora 21, Flavor: Compute, An SSH key you have access to
 * Create Server,
 * When this is complete there will be a command to log into the server, e.g. ``ssh root@${ONMETAL_IP_ADDRESS}``.
 
-To configure any Fedora 20 bare metal machine (e.g. on OnMetal as above)::
+To configure any Fedora 20 or Fedora 21 bare metal machine (e.g. on OnMetal as above)::
 
+   MASTER=<The Buildbot master's hostname or IP address>
+   ONMETAL_IP_ADDRESS=<The IP address of the OnMetal Server>
+   PASSWORD=<The password in "slaves.fedora-20/vagrant.passwords" from the "config.yml" or "staging.yml" file used to deploy the BuildBot master>
    fab -f slave/vagrant/fabfile.py --hosts=root@${ONMETAL_IP_ADDRESS} install:0,${PASSWORD},${MASTER}
 
-Where ``${PASSWORD}`` is the password in ``slaves.fedora-20/vagrant.passwords`` from the ``config.yml`` or ``staging.yml`` file used to deploy the BuildBot master on hostname or IP address ``${MASTER}``.
-
```

Comment:
This seems unrelated to this change. Could you please split it into a separate CL?